src/components/NavigationBar/NavigationBar.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Nav, Navbar, Container} from 'react-bootstrap';
+import styles from './NavigationBar.module.scss';
+
+export const NavigationBar = () => (
```

Comment:
The navigation bar looks unfinished compared to the wireframes, please complete the NavigationBar. The wireframes are linked in the trello card. https://trello.com/c/a281tY6h